admin/settingstree.php

```diff
@@ -0,0 +1,45 @@
+<?php
+// must be run within Dokuwiki
+if(!defined('DOKU_INC')) die();
+
+/**
+ * All DokuWiki plugins to extend the admin function
+ * need to inherit from this class
+ */
+class admin_plugin_dw2pdf_settingstree extends DokuWiki_Admin_Plugin {
+	private $helper = null;
```

Comment:
Reading your reactions, I understand that your plans are bigger than I first had understood. I recognize the large effort you perform sofar. I interested in your concepts for more finegrained export possibilties...(for example I m also involved in the bookcreator plugin, and know that people are interested in a more powerful interface.)
Maybe you experience my reactions sometimes as a bit conservative, but mainly that is because I try to think about how things can be maintained in the future. And therefore it helps often if things are simpler..
I'm trying to perform some experimenting this weekend and look further into this.